torch/distributed/internal_rpc_utils.py

```diff
@@ -5,28 +5,54 @@
 import io
 import pickle
 import traceback
+import threading
+import sys
 
+import torch
+
+_thread_local_tensor_tables = threading.local()
+_thread_local_tensor_tables.recv_tables = []
+_thread_local_tensor_tables.send_tables = []
+
+
+def _tensor_receiver(tensor_index):
+    global _thread_local_tensor_tables
+    return _thread_local_tensor_tables.recv_tables[tensor_index]
+
+def tensor_reducer(obj):
+    global _thread_local_tensor_tables
+    _thread_local_tensor_tables.send_tables.append(obj)
+    tensor_index = len(_thread_local_tensor_tables.send_tables) - 1
+    return (_tensor_receiver, (tensor_index, ))
+
+if sys.version_info >= (3, 0):
```

Comment:
yes, because when internal_rpc_util module is imported by other files, the global variables will be checked at build time. it will fail to build in python2 if not adding the check

Reply:
why six.PY3 is better? trying to be consistent with other rpc files that are using sys.version_info

Reply:
I just find `six.PY3` to be nicer since its much more concise that doing comparisons on sys.version_info.